src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -4,9 +4,9 @@
       <WizardHeader
         class="mb-6"
         :handle-save-draft="handleSaveAndExit"
-        :show-save-button="showSaveButtons"
```

Comment:
Hey @SoLetsDev sorry haven't read the AC on the ticket but we may still need `showSaveButtons` to hide on review page?